caffe2/operators/conv_transpose_op.cc

```diff
@@ -119,6 +119,9 @@ Bias shape:  (1,)
         "adjs",
         "*(type: [int]; default: [])*")
     .Arg(
+		"output_shape",
```

Comment:
weird indent?

Reply:
yes it looks ok locally, but since I use '/t', it is different online...